server/controllers/connectedAccounts.js

```diff
@@ -11,7 +11,7 @@ import { mustBeLoggedInTo } from '../lib/auth';
 const { ConnectedAccount, User } = models;
 
 export const createOrUpdate = (req, res, next, accessToken, data, emails) => {
-  const { utm_source, redirect } = req.query;
+  const { utm_source, redirect } = req.query; // eslint-disable-line camelcase
```

Comment:
For this one an alias could be cleaner than ignoring the warning

```suggestion
  const { utm_source: utmSource, redirect } = req.query;
```